android/app/build.gradle

```diff
@@ -101,9 +101,12 @@ android {
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
-        manifestPlaceholders = [manifestApplicationId: "${applicationId}",
-                        onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
-                        onesignal_google_project_number: "185558680648"]
+        manifestPlaceholders = [
+            manifestApplicationId: "${applicationId}",
+            onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
+            onesignal_google_project_number: "185558680648",
+            gmaps_key: System.getenv("GOOGLE_MAPS_API_KEY") ?: "",
```

Comment:
After playing with this for some amount of time, I have convinced myself this is not an issue. Do you think it is an issue? I think the environment variables are setup correctly.